### PR TITLE
fix preservation file retrieval

### PR DESCRIPTION
**Pull request by jmartin-sul**
* update `Dor::Item#get_preserved_file` usage in `ItemsController#get_preserved_file`.
* remove now-unused `sdr_server`, `sdr_user`, `sdr_pass` from `content` section of dor config files.

closes issue #666 

NOTE: we shouldn't merge this until [dor-services PR #211](https://github.com/sul-dlss/dor-services/pull/211) gets merged and a new dor-services version gets released.  then, i can fix the first commit to point to the newly released dor-services version.  in the meantime, feedback on the other changes would be appreciated.

### CODE REVIEW

**Comment by jmartin-sul** [comment]
> Coverage increased (+0.4%) to 83.*666*% when pulling 7a8c781 on issue*666*-fix-get_preserved_file into 51045a9 on master.

IANAN (i'm not a numerologist) but that's probably a sign, right?

File: Gemfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ gem 'about_page'
 gem 'active-fedora'
 gem 'blacklight', '~> 5.18'
 gem 'blacklight-hierarchy'
-gem 'dor-services', '~> 5.7.0'
+gem 'dor-services', :git => 'https://github.com/sul-dlss/dor-services.git', :branch => 'sdr-gateway-content-endpoint'
```
Comment (ndushay) [inline Gemfile]: yeah, obviously we wouldn't merge it in this state.

**Comment by ndushay** [comment]
:+1:  AFAICT, except for the Gemfile and Gemfile.lock which will ultimately use a new release of dor-services

File: Gemfile [hunk 2]
```diff
@@ -39,7 +39,7 @@ gem 'about_page'
 gem 'active-fedora'
 gem 'blacklight', '~> 5.18'
 gem 'blacklight-hierarchy'
-gem 'dor-services', '~> 5.7.0'
+gem 'dor-services', '~> 5.8', '>= 5.8.0'
```
Comment (ndushay) [inline Gemfile]: you can do this with '~> 5.8.0'

**Comment by jmartin-sul** [comment]
deployed dor-services-app to lyberservices-test, and this branch to argo-stage, and all appears to be well.

**Pull request merged**